test_scripts/API/RAI/ideas_RegisterAppInterface_parameter_steeringWheelLocation.txt

```diff
@@ -0,0 +1,147 @@
+./test_scripts/API/RAI/048_ATF_N_HMI_Does_Not_Respond_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_HMI_doesn't_respond_RAI_WRONG_LANGUAGE
+./test_scripts/API/RAI/049_ATF_N_steeringWheelLocation_IsWrongDataType_WARNINGS_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_wrong_data_type_RAI_WARNINGS
+./test_scripts/API/RAI/050_ATF_N_steeringWheelLocation_IsWrongDataType_RESUME_FAILED_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_wrong_data_type_RAI_RESUME_FAILED
+./test_scripts/API/RAI/051_ATF_N_steeringWheelLocation_IsWrongDataType_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_wrong_data_type_RAI_WRONG_LANGUAGE
+./test_scripts/API/RAI/052_ATF_P_steeringWheelLocation_LEFT_WARNINGS_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_left_RAI_WARNINGS
+./test_scripts/API/RAI/053_ATF_P_steeringWheelLocation_LEFT_RESUME_FAILED_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_left_RAI_RESUME_FAILED
+./test_scripts/API/RAI/054_ATF_P_steeringWheelLocation_LEFT_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_left_RAI_WRONG_LANGUAGE
+./test_scripts/API/RAI/055_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WARNINGS_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_center_fake_param_WARNINGS
+./test_scripts/API/RAI/056_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_RESUME_FAILED_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_center_fake_param_RESUME_FAILED
+./test_scripts/API/RAI/057_ATF_P_steeringWheelLocation_CENTER_response_transfered_fake_param_WRONG_LANGUAGE_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_center_fake_param_WRONG_LANGUAGE
+./test_scripts/API/RAI/058_ATF_P_steeringWheelLocation_LEFT_Resumption_RAI_steeringWheelLocation.lua Check_that_SDL_responds_RAI_with_steeringWheelLocation_from_UI_GetCapabilities_after_mobile_disconnect_connect
+./test_scripts/API/RAI/059_ATF_P_steeringWheelLocation_LEFT_hmi_capabilities_RIGHT_IGN_OFF_IGN_ON_RAI_steeringWheelLocation.lua Check_that_SDL_doesnot_store_steeringWheelLocation_from_UI_GetCapabilities_at_next_ign_cycle
+
+./test_scripts/API/IsReady/UI_IsReady/RAI/001_ATF_P_hmi_capabilities_UI_IsReady_available_false_RAI_SUCCESS.lua Check_that_SDL_gets_steeringWheelLocation_from_hmi_capabilities_when_UI_GetCapabilities_not_sent_because_UI_IsReady_available_false_RAI_SUCCESS
```

Comment:
@aderiabin, we still don't have final organisation of scripts and test sets. I will do this when we have correct structure. For now will leave them in this Test Set, not to be lost.